CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ option(BUILD_TESTING "Enable testing with ctest." ON)
 
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
-find_package(Threads REQUIRED)
+include(FindThreads)
```

Comment:
Just out of curiosity (I'm not a cmake expert, so I'd like to get the chance to learn something).  
What's the difference between the two lines (`find_package` and `include` both of threads module)?

Reply:
find_package just sets some variables and ensures package is present

FindThreads is a module that will respect the THREADS_PREFER_PTHREAD_FLAG variable you added. On my ARM target it seems to be more clever than find_package.